support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -62,4 +65,22 @@ export const tests: Tests = {
 		seed: 2,
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 	},
+	threeTierCheckout: {
+		variants: [
+			{
+				id: 'variant',
+			},
+		],
+		isActive: false,
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		omitCountries: countriesAffectedByVATStatus,
```

Comment:
The new 'norm' for all standard abTest's I presume?

Reply:
I guess the new norm for functionality/products where we won't be supportin countries in the "VAT" list